src/lib/realmdb/RealmDB.js

```diff
@@ -394,6 +385,19 @@ class RealmDB implements DB, ProfileDB {
     return this._profiles().updateOne({ user_id: this.user.id }, { $set: fields })
   }
 
+  /**
+   * Set Field to null
+   * @param field
+   * @param privacy
+   * @returns {Promise<Realm.Services.MongoDB.UpdateResult<*>>}
+   */
+  setFieldToNull(field, privacy) {
+    return this._profiles().updateOne(
+      { user_id: this.user.id },
+      { $set: { [field]: { value: null, display: null, privacy } } },
+    )
+  }
```

Comment:
shouldnt be here
to set  field as null we simply do setProfileFields({key: {value:null...})